app/waf/src/services/trusteddevice.service.ts

```diff
@@ -51,3 +54,57 @@ export class TrustedDeviceServiceProvider
     return getService(this.dataSource);
   }
 }
+
+export class TrustedDeviceManager {
+  private service: TrustedDeviceService;
+
+  constructor(svc: TrustedDeviceService) {
+    this.service = svc;
+  }
+
+  async trust(
+    mgmtIp: string,
+    mgmtPort?: number,
+    username?: string,
+    password?: string,
+  ): Promise<TrustedDevice> {
+    let body: TrustedDeviceRequest = {
+      devices: [
+        {
+          targetHost: mgmtIp,
+          targetPort: mgmtPort || 443,
+          targetUsername: username || 'admin',
```

Comment:
The default value doesn't make sense, I think. We should post an error if the required value not exist. 

In another saying, if the username/password/port is not provided, using default 'admin/admin/443' will not guarantee the later processing. 

We have lots of such no-meaning default value settings in code. I don't think lots of them are reasonable.

Reply:
Removed